core/src/main/resources/reference.conf

```diff
@@ -61,10 +61,12 @@ akka.kafka.consumer {
   # The KafkaConsumerActor will throw
   # `org.apache.kafka.common.errors.WakeupException` which will be ignored
   # until `max-wakeups` limit gets exceeded.
+  # DEPRECATED: No op as waking up consumer is not used anymore
```

Comment:
Shouldn't these settings be removed altogether? Or left there commented out to let people understand that they may remove them from their code?
And add `deprecated` to the `ConsumerSettings` `withXyz` methods.